crates/bevy_sprite/src/sprite.rs

```diff
@@ -16,4 +16,48 @@ pub struct Sprite {
     /// An optional custom size for the sprite that will be used when rendering, instead of the size
     /// of the sprite's image
     pub custom_size: Option<Vec2>,
+    /// [`Anchor`] point of the sprite in the world
+    pub anchor: Anchor,
+}
+
+/// How a sprite is positionned relative to it's [`Transform`](bevy_transform::components::Transform).
```

Comment:
Tiny typo nits: `positioned` and `its`.

Also really excited for this feature :D Thank you!